akka-actor/src/main/scala/akka/io/dns/internal/AsyncDnsManager.scala

```diff
@@ -67,7 +72,13 @@ private[io] final class AsyncDnsManager(val ext: DnsExt) extends Actor
       log.debug("Resolution request for {} from {}", name, sender())
       warnAboutOldProtocolUse(name)
       val adapted = DnsProtocol.Resolve(name)
-      resolver.forward(adapted)
+      val reply = (resolver ? adapted).mapTo[DnsProtocol.Resolved]
+        .map { asyncResolved ⇒
+          log.warning("AYSNCCC " + asyncResolved)
```

Comment:
I think you didn't mean to leave this log in 😄

Reply:
hehe yeah